index.html

```diff
@@ -25,9 +25,26 @@
 
 <!--<div id="overlayGIF"></div> overlayGIF details in main.css-->
     <div id="info-overlay">
+    
         <div id="text">
```

Comment:
Yeah, I think that's a Lexy addition to the css from last year. It only gets used for the overlay text.